Cargo.toml

```diff
@@ -14,6 +14,7 @@ readme = "README.md"
 
 [features]
 serde = ["dep:serde"]
+sort = []
```

Comment:
As this does not add a lot of code and no new dependencies, I don't a feature gate is necessary.

Reply:
I'm just worried about the impact on others.
Haha, I'm not very familiar with the features of cargo yet.

Reply:
The impact of "just adding code" is trivial increase in compilation time. But if they do not call the new methods, they will not even have wait for machine code generation.

Reply:
Exactly, we should avoid creating new feature gates in such cases